packages/resolver/lib/sources/truffle/index.ts

```diff
@@ -13,7 +13,7 @@ export class Truffle implements ResolverSource {
   }
 
   async resolve(importPath: string) {
-    if (importPath === "truffle/DeployedAddresses.sol") {
+    if (importPath === `truffle${path.sep}DeployedAddresses.sol`) {
```

Comment:
This variable comes to us with a backslash?

Reply:
yes. https://github.com/trufflesuite/truffle/commit/bba3e555f710deb11c249e1dc967480f614ac0d7

Reply:
oh interesting! well, there ya go. good catching this!

Reply:
So Truffle now takes all these paths and converts them to `path.sep`? Will this happen when, for instance, someone does a `truffle unbox metacoin` and gets the imports included in the tests which contain "/" in all paths? That's interesting, so going forward Windows users will be able to use whichever one they want?

Reply:
Since you two apparently missed that whole business -- yup, exactly, Windows users can use either.  What we now do is:

1. When resolving imports, we convert all slashes to `path.sep`;
2. When passing things to the Solidity compiler, we convert all `path.sep` to slashes.

This code is part of (1), so everything should be `path.sep`.

Reply:
I had understood that it would be for relative paths and didn't realize they would get converted for Truffle libraries as well. That's why I asked.